src/network-client.spec.ts

```diff
@@ -2,9 +2,31 @@ import { describe, expect, test } from '@jest/globals'
 import { envResetFixture } from '../tests/fixtures/env-fixture'
 import * as algokit from './'
 
-describe('network-clients', () => {
+// Don't spam algonode all the time, remove the `skip` to run these manually
+describe.skip('network-clients', () => {
   envResetFixture()
 
+  describe('Retry', () => {
+    test('Retries indexer calls', async () => {
+      const indexer = await algokit.getAlgoIndexerClient(algokit.getAlgoNodeConfig('testnet', 'indexer'))
+
+      await Promise.all(
+        new Array(70).fill(0).map(async (_) => {
```

Comment:
Unfortunately, this won't kick off the 429 unless you run the test more than once. Upping it to 150 like the other is a good fix.